tests/middleware/use.js

```diff
@@ -719,9 +719,6 @@ test('sets up the outgoing middleware which is aware of update when manually set
 
     const bot = botmaster.bots[0];
     try {
-      await bot.sendMessage(
-        outgoingMessageFixtures.textMessage(), { __update: mockUpdate });
-
```

Comment:
We won't need this first test because __associatedUpdate will be undefined the first time botmaster sends a message to a user.